app/components/draft_scheduled_post/draft_scheduled_post.test.tsx

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {fireEvent, screen} from '@testing-library/react-native';
+import React from 'react';
+
+import {switchToThread} from '@actions/local/thread';
+import {switchToChannelById} from '@actions/remote/channel';
+import {DRAFT_TYPE_DRAFT, DRAFT_TYPE_SCHEDULED} from '@constants/draft';
+import {openAsBottomSheet} from '@screens/navigation';
+import {renderWithIntlAndTheme} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import DraftAndScheduledPost from './draft_scheduled_post';
+
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+jest.mock('@actions/local/thread', () => ({
+    switchToThread: jest.fn(),
+}));
+
+jest.mock('@actions/remote/channel', () => ({
+    switchToChannelById: jest.fn(),
+}));
+
+jest.mock('@screens/navigation', () => ({
+    openAsBottomSheet: jest.fn(),
+}));
+
+jest.mock('@context/server', () => ({
+    useServerUrl: jest.fn().mockReturnValue('https://server.com'),
+}));
+
+jest.mock('@components/draft_scheduled_post_header', () => {
+    const MockHeader = () => null;
+    return MockHeader;
+});
+
+jest.mock('./draft_scheduled_post_container', () => {
+    const MockContainer = () => null;
+    return MockContainer;
+});
```

Comment:
Is this working as expected? I always get confused with all the `esModule` stuff and how imports should be mocked.

Either way... if this is working fine, it can be simplified. 0/5 though.
```suggestion
jest.mock('@components/draft_scheduled_post_header', () => (
    () => null;
));

jest.mock('./draft_scheduled_post_container', () => (
    () => null,
));
```